src/module/Ui/templates/discussion/partials/form-renderer.twig

```diff
@@ -1,8 +1,15 @@
 {% do form.prepare() %}
 {{ form().openTag(form) }}
-{{ formCollection(form.get('subscription')) }}
-<div class="discussion-content-wrapper">
-    {{ formRow(form.get('content')) }}
-</div>
-{{ formSubmit(form.get('start')) }}
+{% for element in form.getElements() %}
+    {% if element.getName() == 'content' %}
+        <div class="discussion-content-wrapper">
+            {{ formRow(element) }}
+        </div>
+    {% else %}
+            {{ formRow(element) }}
+    {% endif %}
+{% endfor %}
+{% for fieldset in form.getFieldsets() %}
+    {{ formCollection(fieldset) }}
+{% endfor %}
```

Comment:
hm, I think we don't really need it right now unless it is critical to get things working